packages/react-composites/tests/browser/chat/FileSharing.test.ts

```diff
@@ -173,4 +173,24 @@ test.describe('Filesharing SendBox Errorbar', async () => {
     await stubMessageTimestamps(page);
     expect(await page.screenshot()).toMatchSnapshot('filesharing-sendbox-file-upload-error.png');
   });
+
+  test('shows file upload in progress error', async ({ serverUrl, users, page }) => {
+    await page.goto(
+      buildUrl(serverUrl, users[0], {
+        useFileSharing: 'true',
+        uploadedFiles: JSON.stringify([
+          {
+            name: 'SampleFile.pdf',
+            extension: 'pdf',
+            url: 'https://sample.com/SampleFile.pdf',
+            progress: 0.5
+          }
+        ])
+      })
+    );
+    await waitForChatCompositeToLoad(page);
+    await sendMessage(page, '');
```

Comment:
I think empty messages aren't allowed. Let's send some non-empty message so that this test doesn't trigger that error reportin g behavior.